content/news/2020-12-24-darktable-3.4.0-released/2020-12-24-darktable-3.4.0-released.md

```diff
@@ -0,0 +1,383 @@
+author: Pascal Obry
+date: 2020-12-24 10:30:23+00:00
+layout: post
+title: darktable 3.4.0 released
+lede: archi.jpg
+lede_author: <a href="http://photos.obry.net">Pascal Obry</a>
+tags: announcement, darktable release
+
+We're proud to announce the new feature release of darktable, 3.4.0!
+
+The github release is here: [https://github.com/darktable-org/darktable/releases/tag/release-3.4.0](https://github.com/darktable-org/darktable/releases/tag/release-3.4.0).
+
+As always, please don't use the autogenerated tarball provided by
+github, but only our tar.xz. the checksums are:
+
+```
+$ sha256sum darktable-3.4.0.tar.xz
+6dd3de1f5ea9f94af92838c0be5ff30fdaa599aa1d737dcb562f9e0b2b2dbdda darktable-3.4.0.tar.xz
+
+$ sha256sum darktable-3.4.0.dmg
+8ce1c8da8e3bd581eb28b532727822f0bdfe86501e643310a8c90ab19862a687 darktable-3.4.0.dmg
+
+$ sha256sum darktable-3.4.0.exe
+29dee565292a4a72443874bb970663d46420f5a9ba74a5acfcb3251a48c86f7c darktable-3.4.0.exe
+```
+
+When updating from the currently stable 3.2.x series, please bear in
+mind that your edits will be preserved during this process, but the new
+library and configuration will not be usable with 3.2.x any more, so
+you are strongly advised to take a backup first.
+
+#### Important note: to make sure that darktable can keep on supporting the raw file format for your camera, *please* read [this post](https://discuss.pixls.us/t/raw-samples-wanted/5420?u=lebedevri) on how/what raw samples you can contribute to ensure that we have the *full* raw sample set for your camera under CC0 license!
+
+- Almost 2146 commits to darktable+rawspeed since 3.2
+- 700 pull requests handled
+- 116 issues closed
+
+## The Big Ones
+
+- The user manual is made available at the same time as the
+  release. Note that this user manual is also the last version as a
+  new project has taken over the lead on this. The goal is to have
+  an up-to-date user manual by using a simpler format
+  to write text for contributors. Feedback or contributions for the
+  new format are welcome at https://github.com/darktable-org/dtdocs.
+
+  Current manual (multilingual):
+
+  https://darktable.gitlab.io/doc/
+
+  New manual (English only):
+
+  https://darktable-org.github.io/dtdocs/
```

Comment:
This has now been published to https://www.darktable.org/usermanual/en/ in preparation for 3.4

Reply:
Fixed, thanks!